lib/webrick-patch.rb

```diff
@@ -0,0 +1,36 @@
+require 'webrick/https'
+
+CIPHERS = 'AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256'
+SSL_VERSION = 'TLSv1_2_server'
```

Comment:
On EL6 with Ruby 1.8.7 this fails: `E, [2015-12-01T15:06:25.054465 #22701] ERROR -- : Error during startup, terminating. unknown SSL method `TLSv1_2_server'.`